### ISSUE TITLE
Add ability to serve client entirely from Storage Provider

### ISSUE THREAD

**Issue by barankyle** [EtherealEngine/etherealengine#7990]
Currently, when we serve the client from the storage provider, it's not 100% served from there. dev.etherealengine.com points to the Kubernetes cluster, so the initial HTML is coming from the cluster; the vite build is just pointing every other built file to come from resources-dev.etherealengine.com. It would be more efficient in some ways to have the "serve client from storage provider" option point dev.etherealengine.com to the storage provider wholesale, and then not need a `resources-<stage>` subdomain for files, but rather have all resources come from the client subdomain.

To summarize, this would have CloudFront, etc. be configured to host as `<stage>.<domain>`, and not use a resources subdomain at all.

**Comment by speigg** [comment]
This would also solve issues with workers being on a different subdomain, right?
